fraud/service.go

```diff
@@ -74,9 +74,12 @@ func (f *ProofService) RegisterProofs(proofTypes ...ProofType) error {
 	return nil
 }
 
-// Start sets the stream handler for fraudProtocolID and starts syncing if syncer is enabled.
+// Start joins fraud proofs topics, sets the stream handler for fraudProtocolID and starts syncing if syncer is enabled.
 func (f *ProofService) Start(context.Context) error {
 	f.ctx, f.cancel = context.WithCancel(context.Background())
+	if err := f.registerProofTopics(getRegisteredProofTypes()...); err != nil {
```

Comment:
This is confusing and I don't really like the api here, but I saw that @Wondertan has already mentioned it. Let's open an issue for it, but yes it shouldn't affect merging now

Reply:
change registering proof will be too much for this pr. I've created an issue https://github.com/celestiaorg/celestia-node/issues/1059 and will implement a new approach in scope of a separate PR.

Reply:
yes, perfect! you can resolve